src/internal.c

```diff
@@ -2186,59 +2186,59 @@ static int DoKexInit(WOLFSSH* ssh, byte* buf, word32 len, word32* idx)
         }
 
         if (ret == WS_SUCCESS)
-            ret = wc_HashInit(&ssh->handshake->hash, ssh->handshake->hashId);
+            ret = wc_HashInit(&ssh->handshake->hash, (enum wc_HashType)ssh->handshake->hashId);
```

Comment:
Similar to above, we could make a new variable and make this cast once.  It may look cleaner and be easier to read.

Reply:
Introduced new variable.